server/repository/repository.go

```diff
@@ -370,6 +371,35 @@ func (s *Server) GetHelmCharts(ctx context.Context, q *repositorypkg.RepoQuery)
 	return repoClient.GetHelmCharts(ctx, &apiclient.HelmChartsRequest{Repo: repo})
 }
 
+// getRepoFromAlias returns the corresponding repo URL if an alias is provided, or the original URL if not
+func (s *Server) getRepoFromAlias(ctx context.Context, repo string, forceRefresh bool) (*appsv1.Repository, error) {
+	var repoUrl string
+	if strings.HasPrefix(repo, "@") {
```

Comment:
I will be a bit picky but the logic is the same as reposerver/repository/repository.go

Could we find a solution to factorize this logic ?